app/mailers/code_buddy_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CodeBuddyMailer < ActionMailer::Base
+  default from: "from@example.com"
+
+  def review_posted(current_user, review)
+    mail to: current_user.email,
+      subject: 'Parking Confirmation'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.